tests/e2e/tests/test_profile_irsa.py

```diff
@@ -57,10 +53,13 @@
 )
 
 from e2e.utils.aws.iam import IAMPolicy
+from e2e.utils.k8s_core_api import patch_configmap, delete_configmap, upload_file_as_configmap
 
 TO_ROOT_PATH = "../../"
 CUSTOM_RESOURCE_TEMPLATES_FOLDER = "./resources/custom-resource-templates"
 KATIB_EXPERIMENT_FILE = "katib-experiment-profile-irsa.yaml"
+KATIB_CONFIG_MAP_PATCH_FILE = TO_ROOT_PATH + "deployment/add-ons/profile-aws-iam-plugin/patch_katib_config_map_sa.yaml"
```

Comment:
if its only used for tests, can we keep it under e2e dir ?

Reply:
done